### PR TITLE
upgraded vueJs to Version 2.3.2

### PR DESCRIPTION
**Pull request by chiangkeith**

### CODE REVIEW

File: src/views/Article.vue [hunk 1]
```diff
@@ -151,6 +152,62 @@
...
+      const categorieName = _.get(categories, [ 0, 'name' ], '')
+      const imageUrl = _.get(heroImage, [ 'image', 'resizedTargets', 'mobile', 'url' ], '')
+      const ogImageUrl = _.get(ogImage, [ 'image', 'resizedTargets', 'mobile', 'url' ], '')
+      const pureBrief = truncate(sanitizeHtml(_.map(_.get(brief, [ 'apiData' ], []), (o, i) => (_.map(_.get(o, [ 'content' ], []), (str) => (str)))).join(''), { allowedTags: [] }), 197)
+      const pureTags = _.map(tags, (t) => (_.get(t, [ 'name' ], '')))
+      const sectionName = _.get(sections, [ 0, 'name' ], '')
+      const topicId = _.get(topics, [ '_id' ], '')
+
+      return {
+        title: truncate(title, 21) + ` － 鏡週刊`,
```
Comment (hcchien) [inline src/views/Article.vue]: can we put the "鏡週刊" to the constant?

File: src/views/Home.vue [hunk 2]
```diff
@@ -122,6 +125,29 @@ export default {
...
+      meta: `
+        <meta name="keywords" content="${SITE_KEYWORDS}">
+        <meta name="description" content="${SITE_DESCRIPTION}">
+        <meta name="twitter:card" content="summary_large_image">
+        <meta name="twitter:title" content="${SITE_TITLE}">
+        <meta name="twitter:description" content="${SITE_DESCRIPTION}">
+        <meta name="twitter:image" content="${SITE_OGIMAGE}">
+        <meta property="fb:app_id" content="${FB_APP_ID}">
+        <meta property="fb:pages" content="${FB_PAGE_ID}">
+        <meta property="og:site_name" content="鏡週刊 Mirror Media">
```
Comment (hcchien) [inline src/views/Home.vue]: put it to the constant.

File: src/views/HomeB.vue [hunk 3]
```diff
@@ -132,7 +133,32 @@ export default {
...
+      meta: `
+        <meta name="keywords" content="${SITE_KEYWORDS}">
+        <meta name="description" content="${SITE_DESCRIPTION}">
+        <meta name="twitter:card" content="summary_large_image">
+        <meta name="twitter:title" content="${SITE_TITLE}">
+        <meta name="twitter:description" content="${SITE_DESCRIPTION}">
+        <meta name="twitter:image" content="${SITE_OGIMAGE}">
+        <meta property="fb:app_id" content="${FB_APP_ID}">
+        <meta property="fb:pages" content="${FB_PAGE_ID}">
+        <meta property="og:site_name" content="鏡週刊 Mirror Media">
```
Comment (hcchien) [inline src/views/HomeB.vue]: constant

File: src/views/HomeO.vue [hunk 4]
```diff
@@ -95,7 +96,32 @@ export default {
...
+      meta: `
+        <meta name="keywords" content="${SITE_KEYWORDS}">
+        <meta name="description" content="${SITE_DESCRIPTION}">
+        <meta name="twitter:card" content="summary_large_image">
+        <meta name="twitter:title" content="${SITE_TITLE}">
+        <meta name="twitter:description" content="${SITE_DESCRIPTION}">
+        <meta name="twitter:image" content="${SITE_OGIMAGE}">
+        <meta property="fb:app_id" content="${FB_APP_ID}">
+        <meta property="fb:pages" content="${FB_PAGE_ID}">
+        <meta property="og:site_name" content="鏡週刊 Mirror Media">
```
Comment (hcchien) [inline src/views/HomeO.vue]: constant

File: src/views/List.vue [hunk 5]
```diff
@@ -318,7 +319,69 @@ export default {
...
+        <meta name="keywords" content="${SITE_KEYWORDS}">
+        <meta name="description" content="${ogDescription}">
+        <meta name="section-name" content="${sectionName}">
+        <meta name="twitter:card" content="summary_large_image">
+        <meta name="twitter:title" content="${title}">
+        <meta name="twitter:description" content="${ogDescription}">
+        <meta name="twitter:image" content="${ogImage}">
+        <meta property="fb:app_id" content="${FB_APP_ID}">
+        <meta property="fb:pages" content="${FB_PAGE_ID}">
+        <meta property="og:site_name" content="鏡週刊 Mirror Media">
```
Comment (hcchien) [inline src/views/List.vue]: constant

File: src/views/Questionnaire.vue [hunk 6]
```diff
@@ -217,41 +218,43 @@
...
+          <meta name="twitter:card" content="summary_large_image">
+          <meta name="twitter:title" content="${_title + ' - 鏡週刊 Mirror Media'}">
+          <meta name="twitter:description" content="${_description}">
+          <meta name="twitter:image" content="${_image}">
+          <meta property="fb:app_id" content="${FB_APP_ID}">
+          <meta property="fb:pages" content="${FB_PAGE_ID}">
+          <meta property="og:site_name" content="鏡週刊 Mirror Media">
+          <meta property="og:locale" content="zh_TW">
+          <meta property="og:type" content="article">
+          <meta property="og:title" content="${_title + ' - 鏡週刊 Mirror Media'}">
```
Comment (hcchien) [inline src/views/Questionnaire.vue]: constant

File: src/views/Search.vue [hunk 7]
```diff
@@ -73,7 +74,34 @@ export default {
...
+      meta: `
+        <meta name="keywords" content="${SITE_KEYWORDS}">
+        <meta name="description" content="${SITE_DESCRIPTION}">
+        <meta name="twitter:card" content="summary_large_image">
+        <meta name="twitter:title" content="${title}">
+        <meta name="twitter:description" content="${SITE_DESCRIPTION}">
+        <meta name="twitter:image" content="${SITE_OGIMAGE}">
+        <meta property="fb:app_id" content="${FB_APP_ID}">
+        <meta property="fb:pages" content="${FB_PAGE_ID}">
+        <meta property="og:site_name" content="鏡週刊 Mirror Media">
```
Comment (hcchien) [inline src/views/Search.vue]: constant

File: src/views/Topic.vue [hunk 8]
```diff
@@ -169,7 +170,45 @@ export default {
...
+      meta: `
+        <meta name="keywords" content="${SITE_KEYWORDS}">
+        <meta name="description" content="${ogDescription}">
+        <meta name="twitter:card" content="summary_large_image">
+        <meta name="twitter:title" content="${title}">
+        <meta name="twitter:description" content="${ogDescription}">
+        <meta name="twitter:image" content="${ogImage}">
+        <meta property="fb:app_id" content="${FB_APP_ID}">
+        <meta property="fb:pages" content="${FB_PAGE_ID}">
+        <meta property="og:site_name" content="鏡週刊 Mirror Media">
```
Comment (hcchien) [inline src/views/Topic.vue]: constant

File: src/views/Article.vue (hunk 1 above)
Comment (chiangkeith) [inline src/views/Article.vue]: yes

File: src/views/Home.vue (hunk 2 above)
Comment (chiangkeith) [inline src/views/Home.vue]: yes

File: src/views/HomeB.vue (hunk 3 above)
Comment (chiangkeith) [inline src/views/HomeB.vue]: yes

**Pull request merged**